### PR TITLE
Homepage styling

### PR DESCRIPTION
**Pull request by renebreslauer**

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -10,6 +10,152 @@
  * files in this directory. Styles in this file should be added after the last require_* statement.
  * It is generally better to create a new file per style scope.
  *
```
Comment (JamesKemp296) [inline app/assets/stylesheets/application.css]: Why did you remove these?

Comment (renebreslauer) [reply app/assets/stylesheets/application.css]: That was part of the scaffold boilerplate for a CSS file. It was commented out, but for some reason it was still throwing an error when I tried to run the server.
